CMakeLists.txt

```diff
@@ -279,7 +279,7 @@ include(CheckLibraryExists)
 find_package(LibUV REQUIRED)
 include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 
-find_package(Msgpack REQUIRED)
+find_package(Msgpack 1.0.0 REQUIRED)
```

Comment:
The version specified in the `pkg_search_module` call shouldn't be needed anymore, as the version check will be performed by the final `find_package_handle_standard_args`.